ui/pages/home/home.component.js

```diff
@@ -861,6 +867,12 @@ export default class Home extends PureComponent {
       showWhatsNewPopup &&
       !showSmartTransactionsOptInModal;
 
+    const showAutoDetectionModal =
+      canSeeModals &&
+      isShowTokenAutodetectModal &&
+      !showSmartTransactionsOptInModal &&
+      !showWhatsNew;
```

Comment:
Is the idea that we want these other models to popup first?

Reply:
yes, smartTransaction modal and what's new modal should come first